packages/modules/imodel-browser/src/containers/iModelGrid/useIModelSort.ts

```diff
@@ -32,14 +32,18 @@ const sortStringValues = (a: string, b: string) => a.localeCompare(b);
 
 export const useIModelSort = (
   iModels: IModelFull[],
-  options?: IModelSortOptions
+  options?: IModelSortOptions,
+  skipSort = false
```

Comment:
This is so dumb lol. I hate it and totally understand why it's needed. It's worth adding a comment that it needed to account for the limitations of react hooks.

Reply:
(dumb that it's needed at all, not directed at you)

Reply:
indeed haha, added the comment